### PR TITLE
Remember address

### LINKED ISSUE #69: Improvement: make app remember last used receiver address
the app does not remember the last used address, and resets it to 192.168.0.1 whenever it's closed. android 13

### PR DESCRIPTION
**Pull request by githubashutoshsoni**
Saving last ip address for sender is saved after app restart. 

fix #69

### CODE REVIEW

**Comment by gavv** [comment]
Hi, please rebase your branch on fresh main (and then force push it). Also, please fix CI (see README regarding spotless).

**Comment by githubashutoshsoni** [comment]
Sure, I think I did it right. Can you please verify again? @gavv ?

**Comment by gavv** [comment]
Seems that you did merge instead of rebase. Please see discussion in #61.

**Comment by githubashutoshsoni** [comment]
Okay, I did the following this time:
rebase remember_address to origin/main
there were commits needed to be pushed to remember_address
did force push in the android popup.
Can you please verify now?

File: build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ buildscript {
 
     repositories {
         google()
-        mavenCentral()
+        maven { url 'https://jitpack.io' }
```
Comment (ortex) [inline build.gradle:6]: why did you change it?

**Comment by githubashutoshsoni** [comment]
let me know if there are any further items that needs to be completed for the pull request @ortex

**Comment by ortex** [comment]
LGTM. Thanks!
